meta/Meta.cpp

```diff
@@ -5424,10 +5411,9 @@ sai_status_t Meta::meta_generic_validation_get(
             // (this will not respect create_only with default)
             if (get_object_previous_attr(meta_key, md) == NULL)
             {
-                std::string key = sai_serialize_object_meta_key(meta_key);
-
                 // XXX produces too much noise
```

Comment:
>// XXX produces too much noise [](start = 16, length = 30)

This is an empty block. Just reverse the if-statement?

Reply:
i want to keep this for future possible debug investigation purpose, compiler will optimize this anyway to single block